pallet-chainlink-feed/src/traits.rs

```diff
@@ -0,0 +1,12 @@
+//! Traits
+use crate::{Config, Round};
+
+pub trait OnAnswerHandler<T: Config> {
+	fn on_answer(feed: T::FeedId, new_data: Round<T::BlockNumber, T::Value>);
```

Comment:
I think we should call this with `RoundData`, because that matches the type the `FeedInterface` returns and provides guaranteed direct access to the `anser`

Reply:
This pallet uses a `try_into` for converting `Round` into `RoundData`, I'm considering this fail of `try_into` should not affect other processes ( because this `try_into` is only for `OnAnswer` ) though it seems it will never fail, So I want to leave the error handling to the users

Reply:
> Looks great already!

I'm confused about this PR because I thought this will be very hard to implement due to yesterday's standup...But I realize there are only a few lines in this PR while I'm trying to implement it